src/GraphODataTemplateWriter/.config/TemplateWriterSettings.json

```diff
@@ -2,28 +2,30 @@
     "AvailableLanguages": [ "Android", "ObjC", "CSharp", "Python", "JavaScript" ],
     "TargetLanguage": "CSharp",
     "Plugins": [ ],
-    "PrimaryNamespaceName": "com",
+    "PrimaryNamespaceName": "oneDrive",
```

Comment:
Doing this PR into a OneDrive branch. For the Graph branches these config changes would be reverted, except for getting rid of the Shared mapping below.